src/components/marginals/Navbar/styles.jsx

```diff
@@ -12,68 +12,62 @@ const fadeDown = keyframes`
     }
 `;
 
-export const Logo = styled.img`
-  height: 50px;
-  width: auto;
-  @media (max-width: 1023px) {
-    height: 40px;
-  }
-`;
-
 export const NavSection = styled.header`
   position: fixed;
   top: 0;
   width: 100%;
   box-shadow: 0 0 16px 0 rgba(255, 255, 255, 0.36);
   background-color: var(--background-primary);
-  padding-top: 1em;
-  padding-bottom: 1em;
   z-index: 40;
-  .navHomeLink {
-    display: inline-flex;
-    align-items: center;
-    text-decoration: none;
-    &:hover {
-    }
-  }
   .navLinkList {
     display: none;
     list-style: none;
   }
   .navLinkItem {
-    margin-right: 32px;
     text-decoration: none;
+    padding: 0px 28px;
+    position: relative;
+    display: inline-block;
     &:last-child {
       margin-right: 0;
     }
     &:hover {
       cursor: pointer;
     }
+    &::before {
+      content: '';
+      position: absolute;
+      width: 0;
+      height: 1.5px;
+      bottom: 0;
+      left: 50%;
+      display: block;
+      background: linear-gradient(88.29deg, #ffb8ff 1.56%, #b9cdff 94.54%);
```

Comment:
Use the variable declared in global CSS.
`
background: var(--brand-gradient);
`